test/asana/helpers/test_extract_task_fields_from_pull_request.py

```diff
@@ -255,12 +258,36 @@ def test_html_body_status_closed_approved_before(self, approved_before_merging):
         ]
         self.assertContainsStrings(actual, expected_strings)
 
+    @patch("src.github.logic.pull_request_approved_before_merging")
+    def test_html_body_status_followup_review(self, approved_before_merging):
+        approved_before_merging.return_value = ApprovedBeforeMergeStatus.NEEDS_FOLLOWUP
+        pull_request = build(
+            builder.pull_request()
+            .author(builder.user("github_test_user_login"))
+            .url("https://foo.bar/baz")
+            .body("BODY")
+            .closed(True)
+            .merged(True)
+        )
+        task_fields = src.asana.helpers.extract_task_fields_from_pull_request(
+            pull_request
+        )
+        actual = task_fields["html_notes"]
+        expected_strings = [
+            "<body>",
+            "incomplete",
+            "approved before merging by a Github user that requires follow-up",
+            "BODY",
+            "</body>",
+        ]
+        self.assertContainsStrings(actual, expected_strings)
+
     @patch("src.github.logic.pull_request_approved_before_merging")
     @patch("src.github.logic.pull_request_approved_after_merging")
     def test_html_body_status_closed_approved_after(
-        self, approved_before_merging, approved_after_merging
+        self, approved_after_merging, approved_before_merging
```

Comment:
The order of the arguments from `@patch` was incorrect, so this test was checking the wrong thing

Reply:
whoops